### ISSUE TITLE
read_excel (calamine engine) Boolean schema override not working

### ISSUE THREAD

**Issue by KingDarule** [pola-rs/polars#17407]
### Checks

- [X] I have checked that this issue has not already been reported.
- [X] I have confirmed this bug exists on the [latest version](https://pypi.org/project/polars/) of Polars.

### Reproducible example

```python
   read_excel('Test.xlsx', engine='calamine', sheet_name='Sheet1', infer_schema_length=None, schema_overrides={'NAME': pl.String, 'IS_ACTIVE': pl.Boolean})
```

### Log output

```shell
_fastexcel.InvalidParametersError: invalid parameters: unsupported dtype: "bool"
```

### Issue description

When reading in an Excel column that is TRUE/FALSE (booleans used within an Excel workbook) and explicitly defining it as a pl.Boolean, it throws an unsupported dtype error. I've had to instead read this in as a pl.UInt8 and cast it to a Boolean after its read in as a workaround.

### Expected behavior

I would expect it to allow me to explicitly define a column as a boolean when reading in the Excel file without throwing an error.

### Installed versions

<details>

```
Replace this line with the output of pl.show_versions(). Leave the backticks in place.
```

</details>
polars 1.0
Also broken in 0.2 as well

**Comment by alexander-beedie** [comment]
@KingDarule: fixed - available in the just-released Polars 1.1.0.
Give it a go?

**Comment by KingDarule** [comment]
It works -- thanks for the quick fix!
